samples/ContainmentSuperVsExtendsSameType.java

```diff
@@ -19,8 +19,10 @@
 @NullMarked
 class ContainmentSuperVsExtendsSameType {
   void x() {
+    // `conflicting_annotations` b/c upper and lower bound of `? super Object` differ.
     // :: error: jspecify_nullness_mismatch jspecify_but_expect_nothing
```

Comment:
Separately, as we go through `expect_nothing` and `but_expect_error` messages, it would be nice to have a comment why a mismatch is expected.
Here, I expect the problem is that `Lib<? super Object>` is not a subtype of `Lib<? extends Object>`, right?

Reply:
Yes, I believe that's right.

And yes, much of the idea of the conformance-test framework is that it lets us replace "error of some kind on this line" with more of an "expected Foo but was Foo?" style.